src/core/core.interaction.js

```diff
@@ -25,15 +25,14 @@ function getRelativePosition(e, chart) {
  * @param {function} handler - the callback to execute for each visible item
  */
 function parseVisibleItems(chart, handler) {
-	var metasets = chart._getSortedVisibleDatasetMetas();
-	var metadata, i, j, ilen, jlen, element;
-
-	for (i = 0, ilen = metasets.length; i < ilen; ++i) {
-		metadata = metasets[i].data;
-		for (j = 0, jlen = metadata.length; j < jlen; ++j) {
-			element = metadata[j];
-			if (!element._view.skip) {
-				handler(element, i, j);
+	const metasets = chart._getSortedVisibleDatasetMetas();
+
+	for (let i = 0, ilen = metasets.length; i < ilen; ++i) {
+		const {index, data} = metasets[i];
+		for (let j = 0, jlen = data.length; j < jlen; ++j) {
+			const element = data[j];
+			if (!element.skip) {
```

Comment:
Why did this line drop `_view`?

Reply:
Doh, noticed this while working with animations (no view/model in that draft). I'll fix that.